int64 thumb_adduction
int64 thumb_flexion
int64 index
int64 middle
int64 ring
int64 little